source/MaterialXGenEssl/EsslShaderGenerator.cpp

```diff
@@ -12,30 +12,21 @@ namespace MaterialX
 {
 
 const string EsslShaderGenerator::TARGET = "essl";
-const string EsslShaderGenerator::VERSION = "300 es"; // Target webgl 2.0
+const string EsslShaderGenerator::VERSION = "300 es"; // Target WebGL 2.0
 
 EsslShaderGenerator::EsslShaderGenerator()
     : GlslShaderGenerator()
 {
     _syntax = EsslSyntax::create();
-    // GL ES specific keywords
+    // ESSL specific keywords
     const StringSet reservedWords = { "precision", "highp", "mediump", "lowp" };
     _syntax->registerReservedWords(reservedWords);
 
-    // Temporary overwrites for THREE js
+    // Temporary overwrites for three.js
     _tokenSubstitutions[HW::T_IN_POSITION] = "position";
     _tokenSubstitutions[HW::T_IN_NORMAL] = "normal";
     _tokenSubstitutions[HW::T_IN_TEXCOORD] = "uv";
     _tokenSubstitutions[HW::T_IN_TANGENT] = "tangent";
-
-    // Register override implementations
-    // <!-- <image> -->
-    registerImplementation("IM_image_float_" + EsslShaderGenerator::TARGET, HwImageNode::create);
```

Comment:
What was this doing?. Something with the texture images?

Reply:
Looking further, I guess something that had to do with the shaders we are deleting? If so, what happened with the `IM_image_float_`. Just want to make sure there is no leftover somewhere

Reply:
I assume this was just to demonstrate how we could override specific nodes. As there are no changes to the base glsl implementation required to work for essl, I removed it again. @bernardkwok added those initially, was there another purpose to those?